src/Microsoft.Performance.SDK/Processing/RequiresCookerAttribute.cs

```diff
@@ -12,83 +11,14 @@ namespace Microsoft.Performance.SDK.Processing
     /// or on individual <see cref="TableDescriptor"/> static properties to indicate that a 
     /// table described by the <see cref="TableDescriptor"/> requires a data cooker.
     /// </summary>
-    [Obsolete("This will be removed by SDK v1.0.0 release candidate 1. Please use RequiresSourceCooker or RequiresCompositeCooker attributes instead.")]
-    public class RequiresCookerAttribute
+    public abstract class RequiresCookerAttribute
```

Comment:
should this be internal? it was public before since it was used by plugins directly